tests/unit_tests/data_steward/tools/create_tier_test.py

```diff
@@ -98,38 +116,74 @@ def test_parse_args(self):
                           self.incorrect_parameter_list_6)
 
         # Tests if incorrect choice for deid_stage are given
-        incorrect_deid_stage_choice_args = [
-            [
-                '--credentials_filepath', self.credentials_filepath,
-                '--project_id', self.project_id, '--tier', self.tier,
-                '--idataset', self.input_dataset, '--release_tag',
-                self.release_tag, '--deid_stage', 'deid_base'
-            ],
-            [
-                '--credentials_filepath', self.credentials_filepath,
-                '--project_id', self.project_id, '--tier', self.tier,
-                '--idataset', self.input_dataset, '--release_tag',
-                self.release_tag, '--deid_stage', 'deid_clean'
-            ]
-        ]
+        incorrect_deid_stage_choice_args = [[
+            '--credentials_filepath',
+            self.credentials_filepath,
+            '--project_id',
+            self.project_id,
+            '--tier',
+            self.tier,
+            '--idataset',
+            self.input_dataset,
+            '--release_tag',
+            self.release_tag,
+            '--deid_stage',
+            'deid_base',
+            '--run_as',
+            self.run_as,
```

Comment:
FYI:  I really dislike this, but the pre-commit hook made all these changes.

Reply:
YAPF is going this, I guess it's fine.